src/components/ui-shell/ui-shell.config.js

```diff
@@ -60,41 +60,61 @@ const sidenav = {
   ],
 };
 
-const switcher = {
```

Comment:
@aledavila Did you really mean removing `switcher` here? There still seems a [reference to `switcher`](https://github.com/IBM/carbon-components/blob/85865312fc3569a189fb82d9bbc9579d7d7a362f/src/components/ui-shell/ui-shell.config.js#L131), which seems a cause of [latest Travis failure](https://travis-ci.org/IBM/carbon-components/builds/430158583).

Reply:
It got removed in the merge accidentally have added it back now. Thanks